pkgs/tools/misc/qarma-for-qt6/default.nix

```diff
@@ -0,0 +1,69 @@
+/*
+qt6 build is failing
+https://github.com/luebking/qarma/issues/41
+
+FIXME broken cmake files in qtbase?
+  #include <QX11Application>
+  QX11Application: No such file or directory
+  should be provided by /nix/store/*-qtbase-6.2.1-dev/include/QtGui/qguiapplication_platform.h
+*/
+
+{ mkDerivation
+, lib
+, fetchFromGitHub
+, wrapQtAppsHook
+, qtbase
+, qmake2cmake
+, cmake
+, qt5compat
+, pkg-config
+, libglvnd, libxkbcommon, vulkan-headers # TODO should be inherited from qtbase
+}:
+
+mkDerivation rec {
+  pname = "qarma";
+  version = "2021-10-05";
```

Comment:
```suggestion
  version = "unstable-2021-10-05";
```

Reply:
its not "unstable" ... the maintainer just does not make new releases https://github.com/luebking/qarma/issues/31